### PR TITLE
Last nif functions

### PR DESCRIPTION
**Pull request by SanjoDeundiak**

### CODE REVIEW

File: implementations/e/applications/ockam_vault_software/mix.lock [hunk 1]
```diff
@@ -9,4 +9,5 @@
   "makeup": {:hex, :makeup, "1.0.3", "e339e2f766d12e7260e6672dd4047405963c5ec99661abdc432e6ec67d29ef95", [:mix], [{:nimble_parsec, "~> 0.5", [hex: :nimble_parsec, repo: "hexpm", optional: false]}], "hexpm", "2e9b4996d11832947731f7608fed7ad2f9443011b3b479ae288011265cdd3dad"},
   "makeup_elixir": {:hex, :makeup_elixir, "0.14.1", "4f0e96847c63c17841d42c08107405a005a2680eb9c7ccadfd757bd31dabccfb", [:mix], [{:makeup, "~> 1.0", [hex: :makeup, repo: "hexpm", optional: false]}], "hexpm", "f2438b1a80eaec9ede832b5c41cd4f373b38fd7aa33e3b22d9db79e640cbde11"},
   "nimble_parsec": {:hex, :nimble_parsec, "0.6.0", "32111b3bf39137144abd7ba1cce0914533b2d16ef35e8abc5ec8be6122944263", [:mix], [], "hexpm", "27eac315a94909d4dc68bc07a4a83e06c8379237c5ea528a9acff4ca1c873c52"},
+  "telemetry": {:hex, :telemetry, "0.4.2", "2808c992455e08d6177322f14d3bdb6b625fbcfd233a73505870d8738a2f4599", [:rebar3], [], "hexpm", "2d1419bd9dda6a206d7b5852179511722e2b18812310d304620c7bd92a13fcef"},
```
Comment (SanjoDeundiak) [inline implementations/e/applications/ockam_vault_software/mix.lock:12]: @mrinalwadhwa should that be here?

Comment (mrinalwadhwa) [reply implementations/e/applications/ockam_vault_software/mix.lock:12]: no, we haven't added telemetry to `ockam_vault_software` yet.

https://github.com/ockam-network/ockam/blob/978bebd779d611ef471b93248d29fa6bb3ed9239/implementations/e/applications/ockam_vault_software/mix.exs#L45-L52

Comment (mrinalwadhwa) [reply implementations/e/applications/ockam_vault_software/mix.lock:12]: oh wait ... yes it should be

Comment (mrinalwadhwa) [reply implementations/e/applications/ockam_vault_software/mix.lock:12]: because I added it to `../ockam` and this lock should be updated

**Review by mrinalwadhwa — APPROVED** [review]

**Pull request merged**